src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,812 @@
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-clipboard" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 4
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+              {{> menu-item-select-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-bars" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+      </ul>
+    {{/menu-group}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-favorites
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-group}}
+      {{#> menu-group-title}}
+        Favorites
+      {{/menu-group-title}}
+      <ul>
+        {{#> menu-list-item}}
+          {{#> menu-item}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 1 
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{> menu-item-action menu-item-action--IsFavorite="true" menu-item-action--IsFavorited="true"}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 3
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+            {{/menu-item-main}}
+          {{/menu-item}}
+          {{> menu-item-action menu-item-action--IsFavorite="true"}}
+        {{/menu-list-item}}
+      </ul>
+    {{/menu-group}}
+    {{> divider}}
+    {{#> menu-group}}
+      {{#> menu-group-title}}
+        All actions
+      {{/menu-group-title}}
+      <ul>
+        {{#> menu-list-item}}
+          {{#> menu-item}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 1
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{> menu-item-action menu-item-action--IsFavorite="true"}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 2
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+              {{> menu-item-select-icon}}
```

Comment:
is this a valid combo? can selectable items serve as something that opens in a new window?

<img width="806" alt="Screen Shot 2020-08-18 at 12 29 56 PM" src="https://user-images.githubusercontent.com/35148959/90545828-d413fd80-e14e-11ea-8592-9fe6d4fc5078.png">

Reply:
No i dont think so, i was just testing out how the spacing would look, but will remove the select icon!